project5/test.in

```diff
@@ -0,0 +1,66 @@
+1 0
+BOSTON
+2 1
```

Comment:
Good job including some small tests.  They test limits, and are easy to check manually.  (It would even be useful to add some more 2 and 3 city cases, with various topologies and bridges.)